test/unit/bin/ember-template-lint-test.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+var exec = require('child_process').exec;
+var assert = require('power-assert');
+var path = require('path');
+
+describe('ember-template-lint executable', function() {
+  describe('basic usage', function() {
+    describe('without any parameters', function() {
+      it('should exit without error and any console output', function(done) {
+        exec('./bin/ember-template-lint', function(err, stdout, stderr) {
+          assert.equal(err, null, 'exits without error');
+          assert.equal(stdout, '', 'doesn\'t print anything to standard output');
+          assert.equal(stderr, '', 'doesn\'t print anything to error output');
+          done();
+        });
+      });
+    });
+
+    describe('given path to non-existing file', function() {
+      it('should exit without error and any console output', function(done) {
+        exec('../../../bin/ember-template-lint app/templates/application-1.hbs', {
+          cwd: './test/fixtures/with-errors'
```

Comment:
These repetitive, relative paths can look a bit edgy - I don't really have the mindset of striving for DRY-ness/sparkling code in tests, too much. They're here, they work, they're simple and readable. Comments/suggestions welcome.